src/Symfony/Component/RateLimiter/Tests/Policy/SlidingWindowTest.php

```diff
@@ -29,7 +29,7 @@ public function testGetExpirationTime()
 
         $data = serialize($window);
         $cachedWindow = unserialize($data);
-        $this->assertNull($cachedWindow->getExpirationTime());
+        $this->assertSame(2 * 10, $cachedWindow->getExpirationTime());
```

Comment:
Now that serialization doesn't change this anymore, I think it would be good to replace line 30 and 31 with `sleep(10)` and check that `$cachedWindow->getExpirationTime()` returns 10 (if I read the discussion correctly, that should be the expected return time)

Reply:
updated